### ISSUE TITLE
No SAML login support, breaks Enterprise SSO options

### ISSUE THREAD

**Issue by teward** [element-hq/element-x-android#3855]
#### What would you like to do
The homeserver provider I use has SAML login enabled, and only SAML login enabled. This is a requisite, and is present in standard Element client but *NOT* in Element X

#### Why would you like to do it?
Required for Enterprise SSO login on some homeservers including the ones I use

#### How would you like to achieve it?
Create/design a SAML login component to the mechanisms to login to Matrix homeservers, etc. in order to allow Element X to be used with SSO-enforced endpoints, which is CURRENTLY not an option.

### Have you considered any alternatives?

The only alternative is standard Element client or other Matrix clients.

### Are you willing to provide a PR?

No

**Comment by ShadowRZ** [comment]
@teward Element X only supports Native OIDC login with https://github.com/element-hq/matrix-authentication-service

Synapse builtin SSO doesn't work with Element X.

**Comment by teward** [comment]
> @teward Element X only supports Native OIDC login with https://github.com/element-hq/matrix-authentication-service
> 
> Synapse builtin SSO doesn't work with Element X.
> 

Obviously. However thats why this is filed as an enhancement request. And given the eight upvotes this has, it seems that this is desired.

**Comment by ShadowRZ** [comment]
@teward That wouldn't happen, as is already explained elsewhere, see https://github.com/element-hq/element-x-android/issues/3949#issuecomment-2558437507 and https://github.com/element-hq/element-x-android/issues/4138#issuecomment-2590389557
